cli/cli_chestxray_tutorial_test.sh

```diff
@@ -13,42 +13,57 @@ wget -P $DIRECTORY https://storage.googleapis.com/medperf-storage/chestxray_tuto
 tar -xzvf $DIRECTORY/sample_raw_data.tar.gz -C $DIRECTORY
 chmod a+w $DIRECTORY/sample_raw_data
 
+
+# the config does not exists still, but is created automatically by any command
+echo "====================================="
+echo "Changing storage to tmp location"
+echo "====================================="
+# this 'move' command is used only for config updates.
+ev medperf storage move -t $MEDPERF_STORAGE
+checkFailed "moving storage failed"
+echo "We have to return the content back manually, so new tmp folder is empty"
+echo "and all the existing data is kept on the old place."
+mkdir -p ~/.medperf
+ev mv -f $MEDPERF_STORAGE/.medperf/* ~/.medperf/
+##########################################################
+
+
 echo "=========================================="
 echo "Creating test profiles for each user"
 echo "=========================================="
-medperf profile activate local
+ev medperf profile activate local
```

Comment:
`ev` is a custom function defined in tests_setup: it echoes command before execution

Reply:
Could we have a more descriptive name for this? Or an easy way to identify where this function is defined and what is it used for? I can see myself getting confused by this in the future. Maybe `eval` or `print_eval`. Might be longer but I'd prefer readability.

Reply:
> Could we have a more descriptive name for this? Maybe `eval` or `print_eval`. Might be longer but I'd prefer readability.

Sure:) IMO `ev` is more readable though. I pushed the fix (renamed `ev` to `print_eval`). Plz look and let me know - if it is better for you, let's keep it longish.